### PR TITLE
Refine codec2.0 sepolicy in user build

### PR DESCRIPTION
**Pull request by TianmiChen**
Signed-off-by: Chen, Tianmi <tianmi.chen@intel.com>

### CODE REVIEW

**Comment by TianmiChen** [comment]
@dyang23 , @ceiba1985 , @zhangyichix, @yuanjunhuang, could you please help take a review?

Thanks,

**Review by dyang23 — APPROVED** [review]
Thanks

File: codec2/msdk-codec2/mediacodec.te [hunk 1]
```diff
@@ -0,0 +1,2 @@
+allow mediacodec proc_uptime:file { read write open };
```
Comment (ceiba1985) [inline codec2/msdk-codec2/mediacodec.te:1]: why also add these rules in graphics/mesa/mediacodec.te?

Comment (TianmiChen) [reply codec2/msdk-codec2/mediacodec.te:1]: It is needed to grant access to capture the crash messages.

File: aafd/codec2_hal_default.te [hunk 2]
```diff
@@ -0,0 +1 @@
+get_prop(codec2_hal_default, vendor_video_codec_prop)
```
Comment (ceiba1985) [inline aafd/codec2_hal_default.te]: vendor_video_codec_prop and codec2_hal_default are declared in different directories(mixin groups), if one of them is disabled, there would be build errors for this. please do this:
1. add "BOARD_SEPOLICY_M4DEFS += module_aafd=true" to device/intel/mixins/groups/aaf/true/BoardConfig.mk
2. remove this line, and add   module_only(`aafd', `
  get_prop(codec2_hal_default, vendor_video_codec_prop)
')  to codec2/msdk-codec2/codec2_hal_default.te, just like wlan/load_iwlwifi/load_iwlwifi.te.

**Review by ceiba1985 — APPROVED** [review]

**Review by dyang23 — APPROVED** [review]

**Review by YuanjunHuang — APPROVED** [review]

**Review by zhangyichix — APPROVED** [review]

**Pull request merged**